src/fusion_power_plant.cc

```diff
@@ -227,9 +259,9 @@ void FusionPowerPlant::OperateReactor() {
 }
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 void FusionPowerPlant::DecayInventories() {
-  //Left empty to quickly check if code builds
-  //tritium_storage.Decay();
-  //tritium_excess.Decay();
+  tritium_storage.Decay();
+  tritium_excess.Decay();
+  sequestered_tritium->Decay(context()->time());
```

Comment:
I think this does the current time by default, just like the resbuf Decay, so you could just call `Decay()` with no arguments here to be like the others.

Reply:
Unfortunately, it does not. The material version of decay requires a time `Decay (int current_time) {` whereas the ResBuf version requires the same time, but with a default `Decay (int current_time = -1) { `. If you take out the `context()->time()` bit you get an error: `decay cannot use default time with NULL context`.

Reply:
I see... that's because `sequestered_tritium` is no tracked.